### PR TITLE
test(annotation): cover distribute_tasks_extensively

### PR DESCRIPTION
**Pull request by UkiStefanovic**
Covered the function distribute_tasks_extensively that already had a few tests, but did not have full coverage. To fix the coverage, I mocked the SPLIT_MULTIPAGE_DOC environment variable.

### CODE REVIEW

**Review by Negoje — COMMENTED** [review]
Seems good

File: annotation/.pre-commit-config.yaml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# https://pre-commit.com
+# https://pre-commit.com/hooks.html
+
```
Comment (cakeinsauce) [inline annotation/.pre-commit-config.yaml:3]: remove, please

File: annotation/tests/test_distribution.py [hunk 2]
```diff
@@ -1041,59 +1041,70 @@ def test_distribute_annotation_limit_50_pages(
 
 @pytest.mark.unittest
 @pytest.mark.parametrize(
-    ["files", "annotators", "extensive_coverage"],
+    ["files", "annotators", "extensive_coverage", "split_multipage_doc_setup"],
```
Comment (cakeinsauce) [inline annotation/tests/test_distribution.py:1044]: list -> tuple

File: annotation/tests/test_distribution.py [hunk 3]
```diff
@@ -1041,59 +1041,70 @@ def test_distribute_annotation_limit_50_pages(
 
 @pytest.mark.unittest
```
Comment (cakeinsauce) [inline annotation/tests/test_distribution.py:1042]: remove this, please

File: annotation/tests/test_distribution.py [hunk 4]
```diff
@@ -1041,59 +1041,70 @@ def test_distribute_annotation_limit_50_pages(
...
             [copy(ANNOTATORS[0]), copy(ANNOTATORS[2]), copy(ANNOTATORS[3])],
             1,
+            "",
         ),
     ],
 )
 @pytest.mark.unittest
 def test_distribution_with_extensive_coverage(
-    files, annotators, extensive_coverage
+    files, annotators, extensive_coverage, split_multipage_doc_setup
```
Comment (cakeinsauce) [inline annotation/tests/test_distribution.py:1093]: add typings, please

File: annotation/tests/test_distribution.py [hunk 5]
```diff
@@ -1041,59 +1041,70 @@ def test_distribute_annotation_limit_50_pages(
...
         ),
         (
             [copy(file) for file in FILES_PARTIAL],
             [copy(ANNOTATORS[0]), copy(ANNOTATORS[2]), copy(ANNOTATORS[3])],
             1,
+            "",
         ),
     ],
 )
 @pytest.mark.unittest
```
Comment (cakeinsauce) [inline annotation/tests/test_distribution.py:1091]: remove this, please

**Review by cakeinsauce — APPROVED** [review]
lgtm!

**Pull request merged**